apps/storybook/src/Tile.test.ts

```diff
@@ -0,0 +1,22 @@
+describe('Tile', () => {
+  const storyPath = 'Core/Tile';
+  const tests = [
+    'Basic',
```

Comment:
just thought if we have lets say a pixel difference (in spacing or smth), would it fail the test or it might be too small diff as the image taken is much bigger than the component.

Reply:
I tested with a small pixel difference: Removed the `.` from "More Info." in `Alert`.

Test failed successfully 👍
![image](https://user-images.githubusercontent.com/9084735/168866692-30cd70eb-cab7-4e24-a47c-c21583efc8a5.png)